token/cli/src/main.rs

```diff
@@ -2128,6 +2138,13 @@ fn main() {
             SubCommand::with_name("gc")
                 .about("Cleanup unnecessary token accounts")
                 .arg(owner_keypair_arg())
+                .arg(
+                    Arg::with_name("del_associated_accounts")
+                    .long("del_associated_accounts")
```

Comment:
nit, use kebab-case for flags 
```suggestion
                    .long("close-empty-associated-accounts")
```